bin/git-delete-tag

```diff
@@ -3,6 +3,16 @@
 # Assert there is at least one tag provided
 test -z $1 && echo "tag required." 1>&2 && exit 1
 
+# Set the default origin
+origin="origin"
+
```

Comment:
```suggestion
```

Reply:
Move the `origin` into the `else` and update.